react-native-adjust.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'react-native-adjust'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.source         = { :git => 'https://github.com/adjust/react_native_sdk', :tag => s.version }
+
+  s.requires_arc   = true
+  s.platform       = :ios, '8.0'
+
+  s.preserve_paths = 'LICENSE', 'README.md', 'package.json', 'index.js'
+  s.source_files   = 'ios/*.{h,m}'
```

Comment:
Hey everyone and thanks for the discussion. Lemme get this right - so you would like us to add new pod to Cocoapods (like we currently have our native iOS SDK in here https://cocoapods.org/pods/Adjust) repository so that you can use iOS RN SDK directly through Cocoapods?